pylint/config/callback_actions.py

```diff
@@ -377,13 +377,13 @@ def __call__(
     ) -> None:
         assert isinstance(values, (tuple, list))
         msgids = utils._check_csv(values[0])
-        try:
-            for msgid in msgids:
+        for msgid in msgids:
+            try:
                 self.linter.disable(msgid)
-        except exceptions.UnknownMessageError:
-            # pylint: disable-next=fixme
-            # TODO: Optparse: Raise an informational warning here
-            pass
+            except exceptions.UnknownMessageError:
+                # pylint: disable-next=fixme
+                # TODO: Optparse: Raise an informational warning here
```

Comment:
> @cdce8p This was caused because in the homeassistant config there is a message that has been removed. This caused us to stop parsing the disable key when we reached it.

Didn't know that. I don't usually check all disable message ids to see if they are still valid.
A warning for it (not just informational) would be nice.